.dotnet/src/OpenAI.csproj

```diff
@@ -72,6 +72,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="8.0.0" PrivateAssets="All" />
-    <PackageReference Include="System.ClientModel" Version="1.1.0-beta.5" />
+    <PackageReference Include="System.ClientModel" Version="1.1.0-beta.6" />
```

Comment:
I'm happy to, but I don't see where the version is set in that script.  Can you point me to where the update should be made?

Reply:
Oh, it seems like Travis made a change to grab the latest one in alphabetical order in the `.nuget` folder? 🤔

```powershell
$systemClientmodelRef = Get-ChildItem -Recurse `
    -Path "$($env:UserProfile)\.nuget\packages\system.clientmodel" `
    -Include "netstandard2.0" | Select-Object -Last 1
```

I actually think this should be "pinned" to match the specific version that we're using here in the OpenAI.csproj. Otherwise, if someone ever downloaded a newer version, it might not match.

Reply:
In other words, I think it should be pinned like this:
```powershell
$systemClientmodelRef = "$($env:UserProfile)\.nuget\packages\system.clientmodel\1.1.0-beta.6\lib\netstandard2.0" `
```

Reply:
SG, copying that in and testing ...